### ISSUE TITLE
missing LICENSE and CHANGELOG on pypi

### ISSUE THREAD

**Issue by ktdreyer** [ceph/chacractl#5]
When I run `setup.py sdist`, I see that the `LICENSE` and `CHANGELOG.rst` are present in my tarball:

```
$ python setup.py sdist
...
$ tar tzf dist/chacractl-0.0.6.tar.gz | egrep "(LICENSE|README|CHANGELOG)"
chacractl-0.0.6/LICENSE
chacractl-0.0.6/README.rst
chacractl-0.0.6/CHANGELOG.rst
```

But these files are missing from the tarball that is on pypi:

```
$ wget http://pypi.python.org/packages/source/c/chacractl/chacractl-0.0.6.tar.gz
$ tar tzf chacractl-0.0.6.tar.gz  | egrep "(LICENSE|README|CHANGELOG)"
chacractl-0.0.6/README.rst
```

What's up with that?

**Comment by ktdreyer** [comment]
Weird, still an issue with v0.0.8. README.rst is present in chacractl-0.0.8.tar.gz; CHANGELOG and LICENSE are not.

**Comment by ktdreyer** [comment]
The reason I see that behavior on my computer is that I had [setuptools-git](https://pypi.python.org/pypi/setuptools-git) installed.

**Comment by bigjust** [comment]
These files are included with the latest tarball 0.0.21, probably been fixed for a while
